test/functional/test-amp-pixel.js

```diff
@@ -116,6 +116,21 @@ describes.realWin('amp-pixel', {amp: true}, env => {
           'https://pubads.g.doubleclick.net/activity;r=111');
     });
   });
+
+  it('should respect referrerpolicy=no-referrer', () => {
```

Comment:
At this moment, I don't think we have such infrastructure ready to test if a request is received on local server. or maybe @rsimha-amp is working on something related?